js/src/helpers/replacementVariableHelpers.js

```diff
@@ -200,3 +200,19 @@ export function mapCustomFields( replaceVars, store ) {
 		...replaceVars,
 	}, "custom_fields" );
 }
+
+/**
+ * Extracts the excerpt from the given content.
+ *
+ * @param {string} content The content.
+ * @param {number} limit   The amount of characters to extract.
+ *
+ * @returns {string} The generated excerpt.
+ */
+export function excerptFromContent( content, limit = 156 ) {
+	// Retrieves the first 156 chars from the content.
```

Comment:
Probably I've missed some internal discussion, but I'm not sure why we're getting the first 156 characters. The WordPress excerpt is based on the number of words (55), not the number of characters. This is also inconsistent with what `WPSEO_OpenGraph` does for the `og:description`, as it fallbacks to `get_the_excerpt()` which returns the first 55 words.

Reply:
I first implemented the WordPress way, but that show some differences with the frontend. Some research shows me that we have some own logic for getting the excerpt from the content. 

I discussed this approach with @jdevalk and he agreed that it was not ideal, but also that doing it this way results in less hassle.